### ISSUE TITLE
last code error Encrypted data

### ISSUE THREAD

**Issue by hyt47** [Jessecar96/SteamDesktopAuthenticator#118]
I was testing your code (last code not the release) (using Visual Studio 2015)
 - the app works fine with unencrypted steam data

If I encrypt my steam data,
  - I don't get another window asking me for passkey to decrypt &  I can't see my accounts in the GUI

- if I restart the program I get a popup asking for encryption passkey & after I click ok > error

```
Unhandled exception has occurred in your application. If u click Continue, the application will ignore this error and attempt to continue. If u click Quit ....
Object reference not set to an instance of an object.

**Details:**
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at Steam_Desktop_Authenticator.MainForm.<UpdateSession>d__37.MoveNext() in F:\Git Hub Projects\v1\SteamDesktopAuthenticator-master\Steam Desktop Authenticator\MainForm.cs:line 455
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Steam_Desktop_Authenticator.MainForm.<UpdateCurrentSession>d__35.MoveNext() in F:\Git Hub Projects\v1\SteamDesktopAuthenticator-master\Steam Desktop Authenticator\MainForm.cs:line 449
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Steam_Desktop_Authenticator.MainForm.<MainForm_Shown>d__8.MoveNext() in F:\Git Hub Projects\v1\SteamDesktopAuthenticator-master\Steam Desktop Authenticator\MainForm.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)


************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework/v4.0.30319/mscorlib.dll
----------------------------------------
Steam Desktop Authenticator
    Assembly Version: 0.3.1.0
    Win32 Version: 0.3.1
    CodeBase: file:///F:/Git%20Hub%20Projects/v1/SteamDesktopAuthenticator-master/Steam%20Desktop%20Authenticator/bin/Debug/Steam%20Desktop%20Authenticator.exe
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
CefSharp
    Assembly Version: 45.0.0.0
    Win32 Version: 45.0.0.0
    CodeBase: file:///F:/Git%20Hub%20Projects/v1/SteamDesktopAuthenticator-master/Steam%20Desktop%20Authenticator/bin/Debug/CefSharp.DLL
----------------------------------------
Newtonsoft.Json
    Assembly Version: 7.0.0.0
    Win32 Version: 7.0.1.18622
    CodeBase: file:///F:/Git%20Hub%20Projects/v1/SteamDesktopAuthenticator-master/Steam%20Desktop%20Authenticator/bin/Debug/Newtonsoft.Json.DLL
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System.Numerics
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Numerics/v4.0_4.0.0.0__b77a5c561934e089/System.Numerics.dll
----------------------------------------
System.Runtime.Serialization
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Runtime.Serialization/v4.0_4.0.0.0__b77a5c561934e089/System.Runtime.Serialization.dll
----------------------------------------
System.Xml.Linq
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Xml.Linq/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.Linq.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
System.Data
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_32/System.Data/v4.0_4.0.0.0__b77a5c561934e089/System.Data.dll
----------------------------------------
SteamAuth
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///F:/Git%20Hub%20Projects/v1/SteamDesktopAuthenticator-master/Steam%20Desktop%20Authenticator/bin/Debug/SteamAuth.DLL
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.

```

**Comment by pipe01** [comment]
Seems like it's caused by the UpdateSession method.

**Comment by Jessecar96** [comment]
Does this still happen?

**Comment by Jessecar96** [comment]
Fixed in ba76b85c16865f13f8d486af96287acd0e41a15e
